scripts-2.0/components/TaskPicker/index.tsx

```diff
@@ -18,13 +18,17 @@ export interface Task {
   children: Task[];
   path?: string[];
   billable: boolean;
+  externalTaskId: string|null;
```

Comment:
jw, nie wiem czy nei warto zrobic dla spojnosc to jako `number` tak samo jak jest `parentId`

Reply:
wyjasnione, to string przeca